kv/txn_coord_sender_test.go

```diff
@@ -35,18 +36,50 @@ import (
 	gogoproto "github.com/gogo/protobuf/proto"
 )
 
+// retryableLocalSender provides a retry option in the event of
+// range splits.
```

Comment:
Expand this comment: "In real-world use, the DistSender is responsible for retrying in the event of range splits, but some tests in this package do not use DistSender and instead rely on local-only retries." Consider adding a TODO to replace retryableLocalSender with DistSender if you think that is feasible (but it may be more trouble than it's worth to hook up everything DistSender needs).